.gitignore

```diff
@@ -1,10 +1,67 @@
-node_modules
-cjs
-.DS_Store
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
 
-yarn-error.log
-.log/
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
 coverage
 
-# jetbrains
-.idea/
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
```

Comment:
We are not using packages related to bower, so its nice to remove them from .gitignore for now. Let's keep it clean rather than adding all possible ignore items.